src/utils/history.js

```diff
@@ -0,0 +1,2 @@
+import { createBrowserHistory } from 'history';
```

Comment:
I kept seeing an error in the console so I looked into the docs and found this. Let me know what you think. 

https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components

Reply:
Yea you figured it out perfectly. I missed this file and will update for the future 🙏